src/components/MissionsTable.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+
+function MissionsTable({ missionName, description }) {
+  return (
+    <tbody>
+      <tr>
+        <td>{missionName}</td>
+        <td>{description}</td>
+        <td className="member-status">
+          <span>NOT A MEMBER</span>
+        </td>
+        <td>
+          <button type="button">Join Mission</button>
+        </td>
+      </tr>
+    </tbody>
+  );
+}
+
+export default MissionsTable;
+
+MissionsTable.propTypes = {
+  missionName: PropTypes.string.isRequired,
+  description: PropTypes.string.isRequired,
+};
```

Comment:
Please kindly wrap this file in a folder, since this is a group project every file should be wrapped inside of a fold this is the best practice and also the correct file structure, for example all the rocket components files should be wrapped in a Rocket folder, same for missions files components and Dragons.

Reply:
@Pamphilemkp Thank you for the feedback to make the code better. I've implemented the change.